assets/blocks/quiz/quiz-block/quiz-timer-promo.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * WordPress dependencies
+ */
+import { ToggleControl, Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ExternalLinkIcon from '../../../icons/external-link.svg';
+
+/**
+ * Quiz block inserter for adding new or existing questions.
+ */
+const QuizTimerPromo = () => {
+	return (
+		<div className="sensei-lms-quiz-block-settings__quiz-timer-promo">
+			<ToggleControl
+				disabled
+				label={ __( 'Quiz Timer', 'sensei-lms' ) }
+				help={ __(
+					'The quiz timer enables you to control how much time a student has to complete the quiz.',
+					'sensei-lms'
+				) }
+			/>
+
+			<p>
+				<a
+					className="sensei-lms-quiz-block-settings__quiz-timer-promo__link"
+					href="https://senseilms.com/pricing/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=quiz_timer"
+					target="_blank"
+					rel="noreferrer"
+				>
+					{ __( 'Upgrade to Sensei Pro', 'sensei-lms' ) }
+					<Icon
```

Comment:
We can use the [`ExternalLink` component](https://developer.wordpress.org/block-editor/reference-guides/components/external-link/) here. It will take care of displaying the icon, so we'll be able to eliminate some extraneous CSS and the SVG.

Reply:
Hm, should I also update the icon in #4809? A potential issue is that I don't use JS there and the [external dashicon](https://developer.wordpress.org/resource/dashicons/#external) looks a bit different.

Reply:
@m1r0 so I think it makes more sense to use the design we have in ExternalLink since I've checked now and it is already used on other places like for example:
<img width="289" alt="image" src="https://user-images.githubusercontent.com/7208249/154188319-35fc9199-3477-4403-ba9a-3f40bde25121.png">

So the new design would be an outlier, maybe we can ask Andrei if he is okay with using the one on the right.

Reply:
> Should I style it or leaving the default is fine?

Yup, since ultimately we want to base our designs on WordPress', seems better to leave it.

> Hm, should I also update the icon in https://github.com/Automattic/sensei/pull/4809? A potential issue is that I don't use JS there and the [external dashicon](https://developer.wordpress.org/resource/dashicons/#external) looks a bit different.

Yeah, this is why I didn't bother mentioning it in your other PR, since that was PHP code and it would have been trickier. I'm fine with leaving it since it's a very minor detail.